src/app/models/abandoned-animal.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Model representing an abandoned animal.
+ * Used for registration by census users.
+ * @author gjimenez
+ */
+export class AbandonedAnimal {
+  id: number | null;
+  species: string | null;
+  sex: string | null;
+  estimatedAge: string | null;
+  physicalCondition: string | null;
+  behavior: string | null;
+  district: string | null;
+  neighborhood: string | null;
+  observations: string | null;
+  latitude: number | null;
+  longitude: number | null;
+  photoBase64?: string | null;
+
+  constructor(values: Partial<AbandonedAnimal> = {}) {
+    this.id = values.id ?? null;
```

Comment:
para cada linea haga el cambio del operador "??" ah "??="

eso para cada linea en los constructores de los objetos

"??" y "??=" son cosas distintas y es necesario usar el operador "??="


ejemplo:

```suggestion
    this.id = values.id ??= null;
```